app/mailers/ft_forms_mailer.rb

```diff
@@ -2,13 +2,14 @@
 
 class FtFormsMailer < ActionMailer::Base
   default from: 'transit-it@admin.umass.edu'
+  default reply_to: 'fieldtrip@umass.edu'
```

Comment:
This ought to be a configurable value (in application.yml) - I'll show you how to do that.